docs/lab/results/result-formats.mdx

```diff
@@ -27,56 +27,67 @@ An example result:
 ## JSON Results
 
 We also return the parsed results in JSON format, so you can use them to generate your own forms.
-For example for a lipids panel:
 
-```bash Get order json results
-curl --request GET \
-     --url {{BASE_URL}}/v3/order/result \
-     --header 'Accept: application/json' \
-     --header 'Content-Type: application/json' \
-     --header 'x-vital-api-key: <your_api_key>' \
-```
+These results are returned in a structured format, which you can find [here](/api-reference/results/get-results).
+
+<Warning>The `results` field, according to the spec, can return either a `list[BiomarkerResult]` or an untyped `dict`. This is due to backwards compatibility, and you can disregard the untyped `dict`</Warning>
```

Comment:
I would be a bit clear on the last part of the sentence. Perhaps "you can disregard the untyped `dic` if you are a new customer"?

Reply:
Existing customers can too, except for Wyndly (as this is a spiriplex only thing). We wouldn't want customers that wont run into this having to account for it IMO